java/src/main/java/io/process/analytics/tools/bpmn/generator/converter/waypoint/GridSearcher.java

```diff
@@ -69,4 +69,30 @@ public boolean isShapeExistAbove(final Position positionFrom) {
                 .anyMatch(p -> p.getY() == positionFrom.getY() - 1);
     }
 
+
+    public BendConfiguration searchEmptyRow(Position positionFrom, Position positionTo, BendDirection searchDirection) {
```

Comment:
the name doesn't describe what the method does. I will refactor this in a new PR